pkg/lib/flavor/model/feature.go

```diff
@@ -14,34 +14,41 @@ type AES_NI struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
-// TXT
-type TXT struct {
-	Enabled bool `json:"enabled"`
-}
-
 // TPM
-type TPM struct {
-	Enabled  bool     `json:"enabled"`
-	Version  string   `json:"version,omitempty"`
-	PcrBanks []string `json:"pcr_banks,omitempty"`
+type HardwareFeature struct {
```

Comment:
We should add backlog to remove the duplicate structures that live here in and in platform-info.